src/api/addressApi.js

```diff
@@ -27,6 +27,7 @@ const addressApi = {
       if (res && res.data) {
         return res.data;
       }
+      return undefined;
```

Comment:
```suggestion
      return;
```

Would this also work?

Reply:
It wouldn't: removing the explicit `undefined` raises an error about an unnecessary `return` on the last line of the function, and removing the `return` breaks the `consistent-return` lint rule that I intended to fix in the first time.

Maybe we can find a better solution at another time, but for now I'd rather not play with code logic.